homeassistant/helpers/service.py

```diff
@@ -708,6 +709,39 @@ async def _handle_entity_call(
         await result  # type: ignore
 
 
+async def integration_service_call(
```

Comment:
This seems like logic that should live inside the ServiceIntegration ?

Reply:
I can move it.